platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PcesBirthRoundMigration.java

```diff
@@ -224,9 +224,9 @@ private static void migrateEvents(
         final Path temporaryFile = LegacyTemporaryFileBuilder.buildTemporaryFile("new-pces-file");
         final SerializableDataOutputStream outputStream = new SerializableDataOutputStream(
```

Comment:
There is a tiny step to make this file fully protobuf compatible: field tags. Every event should be written as a length-encoded protobuf object. Another field is needed for the version. This is it

Reply:
Are there any examples for this I can look at? (might do it in a separate PR)